kafka/protocol/produce.py

```diff
@@ -61,7 +61,6 @@ class ProduceResponse_v4(Response):
     API_VERSION = 4
     SCHEMA = ProduceResponse_v3.SCHEMA
 
-
```

Comment:
why is this dropped? Typically there are two blank lines between class definitions...

Reply:
fixed